amethyst_core/src/transform/components/local_transform.rs

```diff
@@ -1,22 +1,31 @@
 //! Local transform component.
 
-use cgmath::{Matrix3, Matrix4, Quaternion, Vector3};
+use cgmath::{Deg, InnerSpace, Matrix3, Matrix4, Point3, Quaternion, Rotation, Rotation3, Vector3};
+use orientation::Orientation;
 use specs::{Component, DenseVecStorage, FlaggedStorage};
 
 /// Local position, rotation, and scale (from parent if it exists).
 ///
 /// Used for rendering position and orientation.
 #[derive(Clone, Debug, PartialEq, PartialOrd)]
 pub struct LocalTransform {
-    /// Translation/position vector [x, y, z]
-    pub translation: [f32; 3],
     /// Quaternion [w (scalar), x, y, z]
     pub rotation: [f32; 4],
     /// Scale vector [x, y, z]
     pub scale: [f32; 3],
+    /// Translation/position vector [x, y, z]
+    pub translation: [f32; 3],
 }
 
 impl LocalTransform {
+    /// Rotate to look at a point in space (without rolling)
+    pub fn look_at(&mut self, orientation: &Orientation, position: Vector3<f32>) -> &mut Self {
```

Comment:
I'd like to note that with this PR we would expose cgmath types.

Reply:
I'm not sure if that's a good thing.

Reply:
We already expose cgmath in other places as well (see the camera object as an example). However, I also think that mint would be a better idea. I've been thinking about [streamlining the API with traits and Mint](https://github.com/minecrawler/amethyst/blob/feature/enhanceTransformHelpers/amethyst_core/src/movement.rs) and expanding the functionality to global transformation. The question for me would be: Should I put in Mint types here, or do so in the other PR with the traits (if they are acceptable)?

Reply:
I don't really mind as long as it gets done before the release.